command/util.py

```diff
@@ -40,3 +41,22 @@ def escaped_id_to_id(s):
     return re.sub(r"<@(\w+)\|[^>]+>",
                   r"\1",
                   s)
+
+
+def check_credentials(user, team):
```

Comment:
could this be called check_pawmissions instead ? 😸 i think itd be clearer imo since we use credentials to mean other things too

Reply:
Yah sure